tests/ssl-opt.sh

```diff
@@ -3787,6 +3787,25 @@ run_test    "Authentication: server ECDH p256v1, client optional, p256v1 unsuppo
             -c "! Certificate verification flags"\
             -c "bad server certificate (ECDH curve)" # Expect failure only at ECDH params check
 
+requires_config_enabled MBEDTLS_USE_TINYCRYPT
+run_test    "Authentication: DTLS server ECDH p256v1, client required, server goodcert" \
```

Comment:
Minor: p256v1 is a relatively unusual name for this curve. More common names include P-256, secp256r1, or p256 for short - since you're already using p256 in another test, I'd suggest sticking to p256 everywhere. (Here and in the next test.)

Reply:
Done